recipes/themysto/pre-unlink.sh

```diff
@@ -0,0 +1 @@
+"${PREFIX}/bin/themysto" uninstall --sys-prefix
```

Comment:
This is a step to configure the conda-environment jupyter server to use the installed extensions - see https://github.com/Anaconda-Platform/nb_conda/blob/master/conda.recipe/post-link.sh for an analogous step.